skins/oasis/css/core/breakpoints-small-medium-typography.scss

```diff
@@ -20,13 +20,12 @@
 	h2 {
 		font-size: 20px;
 		line-height: 22px;
-
-		.wikia-page-header & {
-			font-size: $default-body-font-size;
-			line-height: $default-body-line-height;
-		}
 	}
 
+	.wikia-page-header h2{
```

Comment:
Can you write down what is the CSS looking like after your changes?
As to me it looks like it would generate exactly the same selector